### DISCUSSION TITLE
iPXE initializing devices stuck - Lenovo L15

### DISCUSSION THREAD

**Discussion by power270** [ipxe/ipxe#1348] [General]
Hi,

I created a debug ipxe and attached the screenshot. The lenovo L15 had the firmware upgraded to the latest. Any suggestions?
![20241126_143011](https://github.com/user-attachments/assets/aed93bed-eb60-48f7-9d48-4711a7526776)

**Comment by NiKiZe** [comment]
Which version of iPXE is this, and have you made sure to try with the latest? (I have had Lenovo T14 machines hanging in a similar fashion, but working with latest git master)

**Reply by power270** [reply]
I compiled it this morning to run the debug.

**Comment by NiKiZe** [comment]
Some older Lenovo devices hanging but working after `git revert  cb95b5b378b1a61d10770965d82dba535b6be242`

Edit: initialy i referred to one of my internal revert commits and not the original cb95b5b378b1a61d10770965d82dba535b6be242

Bug report #1358
